health-monitor/src/index.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import dotenv from 'dotenv';
+import express from 'express';
+import fetch from 'node-fetch';
+import { ApiPromise, WsProvider } from '@polkadot/api';
```

Comment:
The sort plugin seems not to work, @polkadot should be at the top.

Reply:
done